backend/app/infrastructure/routes/user.py

```diff
@@ -152,6 +171,54 @@ def patch(self, user_address: str):
         return {"status": patron_mode_status}, 200
 
 
+@ns.route("/<string:user_address>/antisybil-status")
+@ns.doc(
+    params={
+        "user_address": "User ethereum address in hexadecimal format (case-insensitive, prefixed with 0x)"
+    }
+)
+class AntisybilStatus(OctantResource):
+    @ns.doc(
+        description="Returns user's antisybil status.",
+    )
+    @ns.marshal_with(user_antisybil_status_model)
+    @ns.response(200, "User's cached antisybil status retrieved")
+    def get(self, user_address: str):
+        app.logger.debug(f"Getting user {user_address} cached antisybil status")
+        try:
```

Comment:
we don't have to catch the exception here, it'll be automatically reformatted and returned to the user by Flask Middleware based on the UserNotFound class's attributes;

and you can put logging functionality directly where the Exceptions occurs

Reply:
:1+

Reply:
Fixed